core/src/test/java/org/apache/accumulo/core/clientImpl/TableOperationsHelperTest.java

```diff
@@ -144,10 +144,15 @@ public void removeProperty(String tableName, String property) {
 
     @Override
     public Iterable<Entry<String,String>> getProperties(String tableName) {
+      return getConfiguration(tableName).entrySet();
+    }
```

Comment:
This method can just be deleted now, since the default method exists in the interface.

Reply:
I had deleted this at first but then had put it back because I was unsure. I will fix this.